### PR TITLE
avoid assert and util

### PR DESCRIPTION
**Pull request by calvinmetcalf**

### CODE REVIEW

File: lib/asn1/base/node.js [hunk 1]
```diff
@@ -1,7 +1,16 @@
-var assert = require('assert');
 var Reporter = require('../base').Reporter;
 var EncoderBuffer = require('../base').EncoderBuffer;
 
+function assert(val, msg) {
```
Comment (indutny) [inline lib/asn1/base/node.js]: This clearly does not belong here... What do you think about `minimalistic-assert` module? :) It seems that many of browserify modules may benefit from it.

Comment (calvinmetcalf) [inline lib/asn1/base/node.js]: done, made you an author on npm and collaborator on the repo

**Comment by indutny** [comment]
Woot! You rock!

**Comment by indutny** [comment]
Landed!

**Pull request closed without merging**